DatadogObjc/Sources/SessionReplay/SessionReplay+objc.swift

```diff
@@ -0,0 +1,109 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+#if os(iOS)
+import DatadogSessionReplay
+
+/// An entry point to Datadog Session Replay feature.
+@objc
+public final class DDSessionReplay: NSObject {
+    override private init() { }
+
+    /// Enables Datadog Session Replay feature.
+    ///
+    /// Recording will start automatically after enabling Session Replay.
+    ///
+    /// Note: Session Replay requires the RUM feature to be enabled.
+    ///
+    /// - Parameters:
+    ///   - configuration: Configuration of the feature.
+    @objc
+    public static func enable(with configuration: DDSessionReplayConfiguration) {
+        SessionReplay.enable(with: configuration._swift)
+    }
+}
+
+/// Session Replay feature configuration.
+@objc
+public final class DDSessionReplayConfiguration: NSObject {
+    public var _swift: SessionReplay.Configuration = .init(replaySampleRate: 0)
```

Comment:
suggestion/ This we can hide, no?
```suggestion
    internal var _swift: SessionReplay.Configuration = .init(replaySampleRate: 0)
```

Reply:
Indeed, sorry about that!